thunder/benchmarks/benchmark_litgpt.py

```diff
@@ -819,8 +819,9 @@ def benchmark_main(return_metrics_as_json=False, json_path="", **kwargs) -> None
 
         print(f"Average iter time: {benchmark.perf_metrics['average_iter_time']:.2f} ms")
         print(f"Memory used: {benchmark.perf_metrics['memory_used_GB']:.02f} GB")
-        print(f"Tokens/s: {benchmark.perf_metrics['tokens_per_sec']:.02f}")
-        print(f"Tokens/s/GPU: {(benchmark.perf_metrics['tokens_per_sec']/world_size):.02f}")
+        if "tokens_per_sec" in benchmark.perf_metrics:
+            print(f"Tokens/s: {benchmark.perf_metrics.get['tokens_per_sec']:.02f}")
```

Comment:
`.get` is subscriptable. Maybe a better version would be to condition on tokens_per_sec not to be None `if benchmark.perf_metrics['tokens_per_sec'] is not None:`?

Reply:
Actually benchmark.perf_metrics.get['tokens_per_sec'] returns None if "tokens_per_sec" is not in benchmark.perf_metrics and then there is problem with formatting: `:.02f`.

I can change it to (to be more readable and cover case with tokens_per_sec present but being None)
```
if tokens_per_sec := benchmark.perf_metrics.get('tokens_per_sec') is not None:
            print(f"Tokens/s: {tokens_per_sec:.02f}")
            print(f"Tokens/s/GPU: {(tokens_per_sec/ world_size):.02f}")
```
WDYT?